handlers/comment.js

```diff
@@ -0,0 +1,366 @@
+const express = require('express');
+const { comment: commentController } = require("../controllers");
+const validate = require('../validate');
+const { body, param } = require('express-validator');
+const config = require('../config')
+const jwt = require('jsonwebtoken');
+const comment = express.Router();
+
+function authentication(request, response, next) {
```

Comment:
REMOVE IT. You already implemented the authentication middleware in a dedicated file, and it should be imported and shared where required. DRY